accname-aam/accname-aam.html

```diff
@@ -242,7 +242,7 @@
         <li id="step1">Initialize:  Set the <code>root node</code> to the given element, the <code>current node</code> to the <code>root node</code>, and the <code>total accumulated text</code> to the empty string ("").</li>
         <li id="step2">Compute the text alternative for the <code>current node</code>:
           <ol>
-            <li id="step2A">If the <code>current node</code> is <a class="termref">hidden</a> <strong>and is not</strong> referenced by <code>aria-labelledby</code> or <code>aria-describedby</code>, nor referenced by a native host language text alternative <a class="termref">element</a> or <a class="termref">attribute</a>, return the empty string.
+            <li id="step2A">If the <code>current node</code> is <a class="termref">hidden</a> <strong>and is not</strong> referenced by <code>aria-labelledby</code> or <code>aria-describedby</code>, nor referenced by a native host language text alternative <a class="termref">element</a> (e.g. <code>&lt;label&gt;</code> in HTML) or <a class="termref">attribute</a>, return the empty string.
```

Comment:
Nit: Do we want the angle brackets? From a quick search, I only found one reference to a specific element (`strong`). But it's sans angle brackets. And the ARIA spec is sans angle brackets. But I see the Core AAM has an instance with angle brackets. So &lt;insert shrug here&gt;. Were it me, I'd remove the angle brackets. But if you feel that they should be there, that's fine.